### PR TITLE
chore: validates and corrects Ruby examples

### PR DESCRIPTION
**Pull request by Justintime50**
Validates all the Ruby examples by running them and tweaking items that weren't correct. Adds the easypost-ruby library to the Gemfile file for local testing.

### CODE REVIEW

**Review by nwithan8 — APPROVED** [review]

File: official/docs/ruby/customs_info/create_customs_info.rb [hunk 1]
```diff
@@ -1,26 +1,24 @@
 require 'easypost'
 
-EasyPost.api_key = ENV['EASYPOST_API_KEY']
+EasyPost.api_key = ENV['EASYPOST_TEST_API_KEY']
```
Comment (jchen293) [inline official/docs/ruby/customs_info/create_customs_info.rb:3]:
```suggestion
EasyPost.api_key = ENV['EASYPOST_API_KEY']
```
Why was this changed?

**Review by jchen293 — APPROVED** [review]

File: official/docs/ruby/customs_info/create_customs_info.rb (hunk 1 above)
Comment (Justintime50) [reply official/docs/ruby/customs_info/create_customs_info.rb:3]: Ah dang, left over from a test. I'll fix this in the next PR so we don't have to get stamps again.
